composer.json

```diff
@@ -11,19 +11,19 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "swarrot/swarrot": "^2.1.1|^3.0",
-        "psr/log": "~1.0",
-        "symfony/framework-bundle": "~2.4|~3.0|~4.0",
-        "symfony/console": "~2.4|~3.0|~4.0",
-        "symfony/yaml": "~2.4|~3.0|~4.0"
+        "php": ">=7.2",
+        "swarrot/swarrot": "^3.3",
+        "psr/log": "~1.1",
+        "symfony/framework-bundle": "~4.2",
+        "symfony/console": "~4.2",
```

Comment:
should we actually drop support for the 3.4 LTS ? It is still actively maintained (and used).

Reply:
Definitively not! I changed for `^3.4|^4.1` instead.